packages/hot-module-replacement/client.js

```diff
@@ -179,8 +179,21 @@ function walkTree(pathParts, tree) {
   return walkTree(pathParts, _module);
 }
 
+// Save default browser functions
+const btoa = window.btoa
+const atob = window.atob
+
+// Replace with unicode support
+window.btoa = function utoa(data) {
+  return btoa(unescape(encodeURIComponent(data)));
+}
+
+window.atob = function atou(data) {
+  return decodeURIComponent(escape(atob(b64)));
```

Comment:
I cannot test at the moment, but at a glance; was `b64` meant to be `data`?

Reply:
Yes, my bad) but i don't now how it working for me.

Thanks for review. I remove this extra method. This is enough:
```javascript
function utoa(data) {
  return btoa(unescape(encodeURIComponent(data)));
}
```